pact/policy-manager/policy-manager.md

```diff
@@ -1,11 +1,35 @@
 # Policy Manager
 
-The Policy Manager is a module that maps that maps the function calls of individual policies of the token. With the upgrade of marmalade V2, token creators now can select multiple policies to program the tokens. These policies can include rules for transfers, burns, sales, and more, providing a flexible and customizable way to manage the behavior of tokens on the platform. The token policies is built using the `kip.token-policy-v2` interface, which defines the standard interface that token policies must implement.
+The Policy Manager is a module that maps that maps the function calls of individual policies of the token. With the upgrade of marmalade V2, token creators now can select multiple policies to program the tokens. These policies can include rules for creation, mints, transfers, burns, sales, and more, providing a flexible and customizable way to manage the behavior of tokens on the platform. The token policies is built using the `kip.token-policy-v2` interface, which defines the standard interface that token policies must implement.
 
-This module includes the following key components:
+## Policy Manager and Quotes
+![alt text](../../images/sale_flow.png)
+
+Another main feature that policy manager allows is having one standard for collecting and distributing the fungibles. Previously in marmalade v1, `fixed-quote-policy` was an example of handling the fungible transfers at sales.
+
+### Offer
+Seller can choose to make a quoted offer by adding `quote` information in transaction data. If not present, the sale will proceed without quotes. Note that individual policies can enforce quoted sale by adding their logic in the code. |
+
+When an offer is made with quotes, the policy manager adds the quotes with `quote-manager.add-quote` function. Quote has additional fields, `quote-guards` and `seller-guard`.
+`quote-guards` represent the list of guards that the seller grants the authority to update the quote price. `seller-guard` represents the seller's guard, which can add or remove the guards from the `quote-guards` list.
+
+### Reserve sale at price
 
+In between `offer` and `buy`, we provide a function, `reserve-sale-at-price`. The purpose of the function is to update the quote price and reserve the `buyer` information. This is only necessary when the bidding contract is added as one of the quote guards and would like to process the sale with quote price different than initial quote price.
```

Comment:
I think it's good to make clear that `bidding contract` is just an example. It's not tied to bidding but any type of contract or account that has been granted access to facilitate the sale process outside of marmalade.